full_test.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Enable extended globbing
+shopt -s extglob
+
+SUITE_PASS=true
+
+# Reset the test log
+echo "" > test.log
```

Comment:
Why not just a new file?

Reply:
fixed. It now uses mktemp to create a random file name for each run and reports that to the user.